src/main/java/me/jellysquid/mods/sodium/client/render/chunk/RenderSectionManager.java

```diff
@@ -254,6 +257,7 @@ private void resetLists() {
         }
 
         this.visibleBlockEntities.clear();
+        ShadowRenderer.visibleBlockEntities.clear();
```

Comment:
Is this needed? We're just assigning the list every frame anyways.

Reply:
Sadly, trying to assign the list every frame seems to cause issues with vanilla/when shadows are disabled in my brief testing.